src/gui/Panel.java

```diff
@@ -145,6 +147,9 @@ protected void paintComponent(Graphics g) {
 		case BLACKWINS:
 			writeMessage(g2, "Črn igralec je zmagal.");
 			break;
+		case ALLCOMPUTERS:
```

Comment:
Je to smiselno?
V vsakem primeru zmaga črni/beli; jaz ne vidim potrebe, da posebej obravnavava ta primer.